bot/exts/info/pypi.py

```diff
@@ -17,6 +17,7 @@
 PYPI_COLOURS = itertools.cycle((Colours.yellow, Colours.blue, Colours.white))
 
 ILLEGAL_CHARACTERS = re.compile(r"[^a-zA-Z0-9-.]+")
+NOT_FOUND_DELETE_DELAY = RedirectOutput.delete_delay
```

Comment:
Not sure if it is needed as a wrapper but I think it is good to have it since it describes the purpose of the constant better than `RedirectOutput.delete_delay`.